### PR TITLE
Portfolio: desktop version pull request

### PR DESCRIPTION
**Pull request by Ahsan12356**
- I have created my portfolio using a mobile-first approach
- I have made it responsive for the desktop version too
- I have integrated linters
- I have used media queries in it for making it responsive for 768px of screen wdth
- I have used html and css in it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,111 +37,132 @@ <h1>I'm Ahsan Ghazni Khan!</h1>
         <section class="cards">
```
Comment (Munsa1) [inline index.html:37]: - [x] the section cards are supposed to alternate, in the first card the image is on the left and the text and the right and on the second card the image is on the right and the text is on the left, and so on, kindly fix this as its part of the requirements.

Provided design
image


Your design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Ahsan12356, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Ahsan12356) [reply index.html:37]: Done. Thanks

File: index.html [hunk 2]
```diff
@@ -32,111 +37,132 @@ <h1>I'm Ahsan Ghazni Khan!</h1>
...
+                        <li><p>css</p></li>
+                        <li><p>javaScript</p></li>
+                    </ul>
+                    <div class="btn_see-projects">
+                    <button type="button" class="btn see-projects">See Projects</button>
+                    </div>
+                </div>
             </div>
             <div class="card">
                 <img src="images/first_card_image.png" alt="">
```
Comment (HAMMAS-SALEEM) [inline index.html:61]: ### Your Project Section
cards-images (copy)

### Figma Template
Figma

- [x] You can see in the first image that after adding a width of 75% to the image in the media query your design looks similar to the Figma template. Please add it to your CSS to make your design complete. :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @Ahsan12356 :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following HTML, CSS best practices


Good job on implementing some of the changes requested by the previous reviewer but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Ahsan12356) [reply index.html:61]: but how do I see this effect in my vscode? Should I give 75% of width in my css to this image?

**Review by sumairq — APPROVED** [review]
### Hi @Ahsan12356 @MudasirSherwani @davmba  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**